docs/README.md

```diff
@@ -4,34 +4,38 @@ legacyUrl: https://docs.ipfs.io/
 description: The home page for developer documentation for IPFS, the InterPlanetary File System.
 ---
 
-# IPFS Documentation
+# Educate yourself
 
-**Welcome to IPFS!** We're glad you're here, and we hope this documentation site will help you make the decentralized Internet a better place for humanity. Whether you’re just learning about IPFS or are looking for detailed reference information, here's where to start.
+<p style="font-size:1.65rem; font-weight: 100; line-height: 2rem; margin-top: -15px;">Whether you're a tiny startup or a huge multinational company, you need to learn how to protect yourself and your users from the dangers of a centralized internet. This website contains everything you need to start using the InterPlanetary File System and help future-proof yourself.</p>
 
-IPFS docs are organized into the following main categories:
+## Host a static website on IPFS
 
-## Install IPFS
+Protect your site from a single point of failure by importing it to the IPFS network, and benefit from decentralized infrastructure. If you're new to IPFS, this is an excellent place to start learning how to build on the decentralized web. [Start the tutorial →](/how-to/host-single-page-site/)
```

Comment:
```suggestion
Protect yourself from single-point-of-failure risks and benefit from decentralized infrastructure by hosting your static website using IPFS. If you're new to IPFS, this is an excellent place to start learning how to build on the decentralized web. 

[Start the tutorial →](/how-to/host-single-page-site/)
```
Tightened the language a little and moved the arrow link to its own line for consistency with others. (Note: "IPFS network" is kind of squishy language just in terms of vaguely implying centralization in the network itself ... we may want to avoid that phrase.)